Assets/Scripts/Enviroment/EnvironmentSpawner.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using Unity.Mathematics;
+using UnityEngine;
+using Random = UnityEngine.Random;
+
+public class EnvironmentSpawner : MonoBehaviour
+{
+    [Header("Spawnable Items:")]
+    public GameObject[] environmentParts;
+    public bool areTrees;
+    
+    [Header("Spawn Constraints:")]
+    public float spawnAreaXMin;
+    public float spawnAreaXMax;
+    public float spawnAreaY;
+    public float spawnCooldown;
+    private float currentSpawnCooldown;
+    
+    [Header("Random Rotation:")]
+    public bool randomRotation;
+    public float randomRotationMin;
+    public float randomRotationMax;
+    
+    [Header("Random Scale:")]
+    public bool randomScale;
+    public float randomScaleMin;
+    public float randomScaleMax;
```

Comment:
Moet dit allemaal public zijn of kan het ook gewoon [Serlizedfields] private zijn?
Btw in de header hoef je geen : neer te zetten, ziet er raar uit in de inspector.